pkgs/desktops/arcan/default.nix

```diff
@@ -0,0 +1,42 @@
+{ callPackage, lib, pkgs }:
+
+rec {
+  # Dependencies
+
+  espeak = pkgs.espeak-ng;
+  ffmpeg = pkgs.ffmpeg-full;
+  harfbuzz = pkgs.harfbuzzFull;
```

Comment:
I think we should inherit them and list them as inputs instead of using pkgs.

Reply:
I am merely using them as aliases.